templates/submitted-response.html

```diff
@@ -0,0 +1,53 @@
+{% extends 'layouts/_base.html' %}
```

Comment:
This is only an initial shell. Duplication on the thank you page i.e "Your answers were submitted for ... " will mean this will need to be update as we will want to use the same component. This page also is still in design and needs extra elements adding which were deemed out of scope (buttons) so it will change